lib/.env.example

```diff
@@ -0,0 +1,4 @@
+# Create a file called .env.local and provide these keys.
+
+# Required for Wallet Connect v2
+NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID=12345
```

Comment:
unfortunately everything in `lib` should remain environment-agnostic so all variables must be passed in :-/

Reply:
Ah okay, apologies my bad I should have realised when there was no example env file in the lib directory. I'll remove this file and leave just leave the projectId key in `lib/constants/index.ts`.